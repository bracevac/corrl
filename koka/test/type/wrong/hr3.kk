function poly(xs : list<forall<a> a -> a>)  {
  1
}

function wrong(x) {
  poly(x)
}