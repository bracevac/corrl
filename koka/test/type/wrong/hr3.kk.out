test/type/wrong/hr3.kk(5,16): error: unannotated parameters cannot be polymorphic
 context : function wrong(x) {
 poly(x)
 }
 term : x
 inferred type: list<forall<a> (a) -> a>
 hint : annotate the parameter with a polymorphic type